docs/sources/get-started/install/linux.md

```diff
@@ -78,6 +78,17 @@ sslcacert=/etc/pki/tls/certs/ca-bundle.crt' | sudo tee /etc/yum.repos.d/grafana.
    ```
    {{< /code >}}
 
+1. For beginners, those following tutorials, and debugging: Edit the run configuration to enable the {{< param "PRODUCT_NAME" >}} UI
```

Comment:
It actually used to be as you're suggesting to change, but I made it this on the suggestion of Lisa, who found this more clear - and we didn't want "optional" to get skipped, but to contextualize who/when should do this.

Reply:
It is an optional step though. I'm reluctant to put "beginner" in the description since that's not the primary intent of this UI.... nor is it a tutorial tool. It's a debugging tool for all skill levels.

If you want to contextualize with respect to the tutorial, then the who/when you should do it in the tutorial - and you have done this in the tip in the tutorial step. 

We could link to https://grafana.com/docs/alloy/latest/tasks/debug/ where further explanations are available.